src/__tests__/__utils__/TestConstants.ts

```diff
@@ -0,0 +1,18 @@
+export const AuthenticationTestConstants = {
```

Comment:
Please create groups for similar messages , please have call with me for this comment if any doubt